src/treetensornetworks/matelem.jl

```diff
@@ -0,0 +1,41 @@
+
+#
+# MatElem
+#
+
+struct MatElem{T}
```

Comment:
We could just use `FillArrays.OneElement` for this can get rid of this type: https://juliaarrays.github.io/FillArrays.jl/stable/#FillArrays.OneElement. Not needed for this PR, I added a note about it to #117.

Reply:
Yes, this is for sure. Good to know.